src/lib/services/access-service.ts

```diff
@@ -95,34 +105,30 @@ export class AccessService {
 
     private groupService: GroupService;
 
-    private groupStore: IGroupStore;
-
     private environmentStore: IEnvironmentStore;
 
     private logger: Logger;
 
     private flagResolver: IFlagResolver;
 
+    private eventService: EventService;
+
     constructor(
         {
             accessStore,
             accountStore,
             roleStore,
             environmentStore,
-            groupStore,
         }: Pick<
             IUnleashStores,
-            | 'accessStore'
-            | 'accountStore'
-            | 'roleStore'
-            | 'environmentStore'
-            | 'groupStore'
-        >,
+            'accessStore' | 'accountStore' | 'roleStore' | 'environmentStore'
+        > & { groupStore?: any }, // TODO remove groupStore later, kept for backward compatibility with enterprise
```

Comment:
boy scouting: removing groupStore in favor of groupService to depend only in one of the two